### DISCUSSION TITLE
Is there a way to add a theme to the Hyde sddm theme menu?

### DISCUSSION THREAD

**Discussion by gabe-toledo** [prasanthrangan/hyprdots#1755] [General]
Ive got a few custom sddm themes I want to try out for my login page. I moved the themes to the `/usr/share/sddm/themes` directory but it seems to only seems to show the same 2 when I try to change it via running the command `Hyde sddm theme`. I also tried changing the theme manually in the `/usr/lib/sddm/sddm.conf.d/default.conf` but I think its getting overwritten somehow because its not actually changing.
![image](https://github.com/user-attachments/assets/10b92edc-f4d8-4e56-8b37-837f2dc7f26d)

**Comment by kRHYME7** [comment]
kdesetting conf file should exist on that theme directory for the CLI to recognize it. checkout Candy and Corner themes and copy the kde settings file.

I forgot the exact file name but it should have kde in it. 

Inside that file is the theme name, so after you copy the conf file rename the Theme section with the exact Sddm theme name.

**Comment by thyeun** [comment]
@gabe-toledo Follow the below change.

For changing theme for ```sddm```
- Go to ```/etc/sddm.conf.d/```
- Edit the file ```kde_setting.conf```
- Change the theme accordingly. 

For add-in new themes for ```sddm```
- Go to ```/usr/share/sddm/themes/```
- Put in your new theme.
